### DISCUSSION TITLE
Bandwidth limit changed to byte

### DISCUSSION THREAD

**Discussion by huiyouyongdemao** [juicedata/juicefs#669] [Ideas]
**What would you like to be added**:
It is hoped that the bandwidth limit calculation unit can be changed to byte, which is currently the bandwidth limit calculation unit Mbps.
byte changed to Mbps
**Why is this needed**:
Better control

**Comment by davies** [comment]
For network bandwidth, we usually use Gbps or Mbps, not bytes.

**Comment by davies** [comment]
@huiyouyongdemao could you say more on why this change is better to user?
